plugins/kontent-plugin/docs/setup.md

```diff
@@ -0,0 +1,12 @@
+1. **Create an empty Kontent project**
```

Comment:
If you have documentation / instructions on how to create a Kontent project I think it would be worth linking here. Even if it is as simple as a link to `/sign-up`: https://app.kontent.ai/sign-up.

Additionally, I think this first point belongs in a `quick-start-intro.md` file similar to what we have for Contentful: https://github.com/gatsbyjs/homepage-starters/blob/1aa006dc2bab02bf4cbd45b27df2b598c69762af/plugins/contentful-plugin/docs/quick-start-intro.md#L1

Reply:
Furthermore, it would be great to see some more detail in `.md` files like we have put together in other plugins.

Specifically, `quick-start-intro.md` which sets some context for the user. A `custom-sections.md` which would detail how one would go about adding a new "Homepage Banner" to the Kontent site. And a `preview.md` with instructions for getting Gatsby Cloud Previews configured.

There's also possibly more there if you would like to detail how to do some blog based content. But that's not necessarily expected.

Reply:
Oh and after you have all the different `.md` files created, you can run `yarn create-readmes` from the root of the project and it will assemble those separate files into a unified `README.md`.